scorex-basics/src/main/scala/scorex/network/HistorySynchronizer.scala

```diff
@@ -138,12 +142,13 @@ class HistorySynchronizer(application: Application) extends ViewSynchronizer wit
             if (idx == 0) {
               val toProcess = updBlocks.takeWhile(_._2.isDefined).map(_._2.get)
               log.info(s"Going to process ${toProcess.size} blocks")
-              toProcess.find(bp => !processNewBlock(bp, local = false)).foreach { case failedBlock =>
-                log.warn(s"Can't apply block: ${failedBlock.json}")
-                if (history.lastBlock.uniqueId sameElements failedBlock.referenceField.value) {
-                  connectedPeer.handlerRef ! PeerConnectionHandler.Blacklist
-                }
-                gotoSyncing()
+              toProcess.find(!processNewBlock(_, local = false)).foreach {
```

Comment:
Just a code formatting, no changes in the functionality. If you decide to rollback the change, I would ask to keep  the placeholder ("_") in lieu of the "bp =>" - that was the main reason not to rollback it.